docs/development_suite/identity-and-access-management/service-account-management.md

```diff
@@ -0,0 +1,158 @@
+---
+id: service-account-management
+title: Manage Service Accounts
+sidebar_label: Manage Service Accounts
+---
+
+Mia-Platform Console allows the creation of Service Accounts, which are typically used for automated processes or to allow a service to access resources on behalf of multiple Users.  
+
+Note that new Service Accounts can only be created at Company level. For each Project or Environment they will be visible in the Identities section.
```

Comment:
should be removed the whole line, since the information may be misleading: as a Company Owner I can create a Service Account inside the Project Settings (as for traditional Users)